### DISCUSSION TITLE
Website Launch - MakerForge.tech

### DISCUSSION THREAD

**Discussion by danic85** [makerforgetech/modular-biped#58] [Announcements]
Those of you who have been following the project for a while may have realised this already, but I'm not great at build videos. The process is very slow and often takes a lot longer than I'd like it to. Similarly, using platforms like Instagram to post updates means that I need to keep the content limited to the format of the platform, which often lacks the detail I'd like to add.

Blog posts on the other hand, and written articles in general, are easy for me to create quickly and efficiently, and often more useful to reference than videos alone would be. Previously I haven't had a platform to share this kind of content, besides the wiki.

Therefore, to help me share content that I consider valuable, I have created a new standalone website [Maker Forge](https://www.makerforge.tech). This site will include progress updates, guides, and news from the wider maker community.

The goal of this project (and any others I work on) is to inspire people to get involved in STEM and robotics in general, and to give them tools and knowledge to enable that. I hope that this website will help further that goal and work as a central location for anyone looking for information about this project in future.

The GitHub repo and discussion area won't be affected by this, and will continue to be maintained alongside the static content at Maker Forge.

Take a look and let me know what else you'd like to see on there!   

[Maker Forge](https://www.makerforge.tech)

**Comment by WhoIsMrSentry** [comment]
I examined the site in detail. It is very nice.👏 Can you add send notifications feature if added a blog post send the notification ?

**Comment by danic85** [comment]
I'm not sure as its just a github pages site, but I'll probably be sharing the new posts on social media to make people aware.


________________________________
